README.md

```diff
@@ -58,45 +55,9 @@ curl \
 
 ## Deploying
 
-You shouldn’t need to deploy this manually because there’s a pipeline setup in Jenkins. If you do want to deploy it manually, you'll need the notify-credentials repo set up locally.
+You shouldn’t need to deploy this manually because there’s a pipeline setup in Concourse. If you do want to deploy it manually, you'll need the notify-credentials repo set up locally.
 
 ```shell
 make (sandbox|preview|staging|production) upload-to-dockerhub
 make (sandbox|preview|staging|production) cf-deploy
```

Comment:
I removed the sandbox target from the Makefile recently but forgot to remove references to it from the README, so could take out anything that mentions the sandbox environment from here since we're tidying it up

Reply:
Yes, I can do this.